modules/oe_subscriptions_anonymous/src/MailTemplate/SubscriptionCreate.php

```diff
@@ -81,7 +81,11 @@ public function prepare(array &$message, array $params): void {
       '@cancel_link' => $cancel_link,
     ];
 
-    $body = $this->t("@entity_link \r\n @confirm_link \r\n @cancel_link", $variables);
+    $body = $this->t("Thank you for showing interest in keeping up with the updates for @entity_link! \r\n
+    Click the following link to confirm your subscription: @confirm_link \r\n
+    If you no longer wish to subscribe, click on the link bellow: @cancel_link \r\n
+    If you didn't subscribe to these updates or you're not sure why you received this e-mail, you can delete it. \r\n
+    You will not be subscribed if you don't click on the confirmation link above.", $variables);
```

Comment:
The white space before the string makes the mail look off space. Let's drop it. You might need to ignore coding standards for these lines.

Reply:
Removed extra spaces